src/main/java/no/nav/veilarbaktivitet/oppfolging/oppfolginsperiode_adder/AdderDao.java

```diff
@@ -0,0 +1,57 @@
+package no.nav.veilarbaktivitet.oppfolging.oppfolginsperiode_adder;
+
+import lombok.RequiredArgsConstructor;
+import no.nav.veilarbaktivitet.person.Person;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Repository;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+@Repository
+@RequiredArgsConstructor
+public class AdderDao {
```

Comment:
Jeg er litt usikker på hva jeg tenker om navnet. Burde vi bruke et norsk ord i stedet for `Adder`? 🤔